indico/modules/events/persons/controllers.py

```diff
@@ -0,0 +1,95 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from collections import defaultdict
+
+from sqlalchemy.orm import joinedload
+
+from indico.core.db.sqlalchemy.principals import PrincipalType
+from indico.modules.events.models.persons import EventPerson
+from indico.modules.events.models.principals import EventPrincipal
+from indico.modules.events.persons.views import WPManagePersons, WPManagePendingPersons
+from indico.web.flask.util import url_for
+from MaKaC.webinterface.rh.conferenceModif import RHConferenceModifBase
+
+
+class RHPersonsList(RHConferenceModifBase):
+
+    CSRF_ENABLED = True
+
+    def _process(self):
+        event_persons_query = (self.event_new.persons
+                               .options(joinedload('event_links'), joinedload('contribution_links'),
+                                        joinedload('subcontribution_links'), joinedload('session_block_links'))
+                               .all())
+        persons = defaultdict(lambda: {'session_blocks': defaultdict(dict), 'contributions': defaultdict(dict),
+                                       'subcontributions': defaultdict(dict)})
+        for event_person in event_persons_query:
+            if event_person.is_principal_pending:
```

Comment:
I don't think this is correct! If you add an event person without submission rights they shouldn't be considered pending. Actually, I wonder if we shouldn't simply show all persons here, even if they do have extra rights and just an email principal. After all, they are an event person.

The point of the "pending" list is more to give you an easy way to contact those people